build.sbt

```diff
@@ -26,22 +26,25 @@ checksums in update := Nil
 
 libraryDependencies ++= Seq(
 
-  "org.apache.spark" %% "spark-core" % testSparkVersion.value % "provided" force(),
```

Comment:
so, do you mean that we still keep the force() here? does this only apply for the spark part?